.gitignore

```diff
@@ -15,4 +15,5 @@ bin/
 
 /text-ui-test/ACTUAL.txt
 text-ui-test/EXPECTED-UNIX.TXT
-data/
```

Comment:
Would it be better to just gitignore the entire folder rather than individual sub-folders?

Reply:
It affects the data folder in the src/test too, hence I made it specific.